### PR TITLE
Some answers are changed

### PR DESCRIPTION
**Pull request by aprazova**

### CODE REVIEW

File: Tests/UK/Test.cpp [hunk 1]
```diff
@@ -111,862 +111,864 @@ int main()
...
 	q12.AddOption("Юнит „уайт бокс“ тестване", 1);
 	q12.AddOption("Интеграционно ", -1);
 	q12.AddOption("Тестване за приемане", -1);
-	
+
 	Question q13("Кога тестов процес може да спре?");
 	q13.AddOption("Когато са отстранени всички дефекти в софтуерната система ", -1);
 	q13.AddOption("Когато е достигнато определено ниво на тестово покритие", 0.5);
 	q13.AddOption("Когато е достигнато определено ниво на надеждност", 0.5);
-	q13.AddOption("Когато всички тестови сценарии са изпълнени успешно", -1);
```
Comment (Mapmo) [inline Tests/UK/Test.cpp:174]: why so?

File: Tests/UK/Test.cpp [hunk 2]
```diff
@@ -111,862 +111,864 @@ int main()
...
+	exam.push_back(q127);
+	exam.push_back(q128);
+	exam.push_back(q129);
+	exam.push_back(q130);
+	exam.push_back(q131);
+	exam.push_back(q132);
+	exam.push_back(q133);
+
+
+	system("chcp 65001  > nul");
```
Comment (Mapmo) [inline Tests/UK/Test.cpp]: why so?

File: Tests/UK/Test.cpp (hunk 1 above)
Comment (aprazova) [reply Tests/UK/Test.cpp:174]: Opps, my mistake.

File: Tests/UK/Test.cpp (hunk 2 above)
Comment (aprazova) [reply Tests/UK/Test.cpp]: To fix the problem with encoding, when you use Code blocks. You can delete it.

**Review by Mapmo — APPROVED** [review]

**Pull request merged**